cdm/core/src/test/data/ncml/enhance/testAddToClassifier.ncml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 1998-2020 John Caron and University Corporation for Atmospheric Research/Unidata
+  ~  See LICENSE for license information.
+  -->
+
+<netcdf xmlns="http://www.unidata.ucar.edu/namespaces/netcdf/ncml-2.2" enhance="all">
+
+    <variable name="class_specs" shape="17" type="double">
+        <attribute name="Cat1" type="string"  value="NaN 0 0" />
```

Comment:
I think each of these attributes needs to be named `range_cat` as defined above to make `creasteFromVariable` work

Reply:
If all of them are called range_cat, it will only load the first one. So I named the first one range_cat and then for all other categories range_cat2, range_cat3... Let me know if there is a better way